_posts/2020-04-17-quarkus-mutual-tls.adoc

```diff
@@ -0,0 +1,405 @@
+---
+layout: post
+title: 'Quarkus - Makes Mutual TLS Easy.'
+date:   2020-04-17 00:00 +0100
+author: mkouba
+synopsis: Quarkus - Mutual TLS Authentication.
```

Comment:
same here, The synopsis is a small sentence in the blog list https://quarkus.io/blog/ before the read more button. Maybe something more actionable e.g. Let's learn how to enable mutual TLS in a Quarkus app.